google_guest_agent/diagnostics.go

```diff
@@ -89,6 +89,7 @@ func (d *diagnosticsMgr) disabled(os string) (disabled bool) {
 }
 
 func (d *diagnosticsMgr) set() error {
+	logger.Infof("export-logs requested")
```

Comment:
Logs should be complete sentences, so capitalization and punctuation.

Also we should stick with the diagnostics naming as this is not a generic log exporter, or just not have this prefixed as no other manager has prefixed logs (although that's not a horrible idea)
so maybe:
"Diagnostics: logs export requested." or "Logs export requested."
"Diagnostics: collecting logs from the system:" or "Collecting logs from the system:"

Reply:
Done